### PR TITLE
Update Open MPI hybrid how-to

### PR DESCRIPTION
**Pull request by gjbex**
The command line option mentioned for Open MPI is deprecated.  Update to current ones.

### CODE REVIEW

File: source/software/hybrid_mpi_openmp_programs.rst [hunk 1]
```diff
@@ -254,19 +254,20 @@ Essential elements of our job script are:
...
    module load foss
    #
    export OMP_NUM_THREADS=5
    #
    export OMP_PROC_BIND=true
+   
+   MPI_NUM_PROCS=$(( $PBS_NP/$OMP_NUM_THREADS ))
    #
-   mpirun -cpus-per-proc $OMP_NUM_THREADS ./hybrid_mpi
+   mpirun --np $MPI_NUM_PROCS  --map-by socket:PE=$OMP_NUM_THREADS  --bind-to core  ./hybrid_mpi
```
Comment (MaximeVdB) [inline source/software/hybrid_mpi_openmp_programs.rst:268]: The (deprecated) ``-cpus-per-proc <num>`` option is still mentioned in the paragraph above -- could you also adapt that part?

File: source/software/hybrid_mpi_openmp_programs.rst [hunk 2]
```diff
@@ -254,19 +254,20 @@ Essential elements of our job script are:
...
+
 
-Advanced issues
-~~~~~~~~~~~~~~~
 
 Open MPI allows a lot of control over process placement and rank
```
Comment (MaximeVdB) [inline source/software/hybrid_mpi_openmp_programs.rst:272]: Is it worth to recommend the "--report-bindings" option here?
It is quite handy for double-checking.

File: source/software/hybrid_mpi_openmp_programs.rst (hunk 1 above)
Comment (gjbex) [reply source/software/hybrid_mpi_openmp_programs.rst:265]: True, modified.

File: source/software/hybrid_mpi_openmp_programs.rst (hunk 2 above)
Comment (gjbex) [reply source/software/hybrid_mpi_openmp_programs.rst:269]: It's mentioned below.

**Pull request closed without merging**